cluster/gce/util.sh

```diff
@@ -342,7 +364,8 @@ function upload-tars() {
   fi
 }
 
-# Detect minions created in the minion group
+# Detect minions created in the minion group. Finds both Linux and Windows
+# minion nodes.
```

Comment:
Nit that has nothing to do with the change in this PR, but since you're changing this line: can we stop referring to minions. eg:

```Detect Linux and Windows nodes created in the instance group.```

Reply:
Done.